homeworks/alina.yaremenko_AlinaYaremenko/3-functions/functions.js

```diff
@@ -0,0 +1,20 @@
+const OIL = 926;
+const WATER = 1000;
+const MERCURY = 1355;
+const TANK_VOLUME = 1;
+
+const OIL_COUNT = 3;
+const WATER_COUNT = 5;
+const MERCURY_COUNT = 2;
+
+const getWeight = (weight, liquid, count) => {
+    const result = weight * liquid * count;
+    return result;
```

Comment:
You don't need this extra variable. You could simplify it like this:
```suggestion
    return weight * liquid * count;
```